source/frontend/src/assets/scss/colors.scss

```diff
@@ -1,55 +1,12 @@
-$primary-color: #003366;
 $primary-hover-color: #335c85;
-$primary-light-color: #38598a;
 $secondary-variant-color: #428bca;
-$light-variant-color: #606060;
-$dark-variant-color: #121212;
-$text-color: #494949;
-$form-text-color: #212529;
-$form-control-text-color: #495057;
-$form-background-color: #d9d9d9;
-$dropdown-background-color: #32507c;
-$accent-color: #fcba19;
-$light-accent-color: #fcd573;
-$sres-icon-color: #172b4d;
-$danger-color: #d8292f;
+$border-outline-color: #606060;
+$form-background-color: #ced4da;
+$warning-background-color: #fef1d8;
```

Comment:
I liked the reduced amount of colors. However, I would prefer that we have a set of color names, like "light-red", instead of the uses for it. We can create variables that use the color, $border-outline-color: $light-grey.

Reply:
In a nutshell, the color palete should be independent from the uses.

Reply:
The reason I went with this, is that is following the pattern set by the bctokens repository (see the link that I posted above).

Reply:
To clarify - I understood that Manuel was suggesting that we create a few variables at the top of this file like so:

```scss

// This is our PIMS palette
$gray-100: #606060;
$gray-200: ...
$gray-300: ...
$red-100: ...
$red-200: ...

...

// This is our color usage throughout PIMS
$border-outline-color: $gray-100;
$some-other-stuff-color: $gray-100; 
```

I like this idea to separate the actual "color palette" from its usage